src/components/ConnectWallet/Wallets/ZapiOS.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <connection-details name="Zap (iOS)">
+    <step-list>
+      <step>
+        Open the Zap app on your iPhone.
+      </step>
+      <step>
+        Tap <span class="font-weight-bold">"Connect"</span> on the welcome screen.
+      </step>
+      <step>
+        Scan this QR Code (click to enlarge)
+        <qr-code
+          :value="urls.lnd.restTor"
+          :size="300"
+          class="qr-image mt-2"
+          showLogo
+          @click="$emit('showQrModal', urls.lnd.restTor.replace(/cert=(.*)&/gm,''))"
```

Comment:
The value in the modal is stripping out the cert param, but the value in non-modal QR is not.

The modal on click should probably be built in to the `<qr-code>` component. It should inherit the same value and be toggled on with `enlarge-on-click="true"` or something similar.

That way we don't have to duplicate the showQrModal code for every QR that needs it. It also means we have a single canonical place to set the value for both QRs to prevent bugs where one value gets updated but the other doesn't, like what happened here.